ibis/bigquery/compiler.py

```diff
@@ -265,6 +265,12 @@ def _timestamp_op(func, units):
     def _formatter(translator, expr):
         op = expr.op()
         arg, offset = op.args
+        if not isinstance(offset, ir.IntervalValue):
+            raise com.UnsupportedOperationError(
+                'Binary operations between two timestamps or dates '
+                'are not yet supported in bigquery backend'
+            )
```

Comment:
Disable just for short term. 

Should we create [separate operations](https://github.com/ibis-project/ibis/blob/master/ibis/expr/operations.py#L2560) for 
- `timestamp - timestamp` 
- `timestamp - interval`
- `date - date`
- `date - interval`

That might be more convenient.

Reply:
Yep, those should be separate

Reply:
We can choose which one to construct inside the functions in `api.py`

Reply:
How about:
* `TimestampDiff`
* `TimestampSub`
* `DateDiff`
* `DateSub`

Reply:
I was considering diff and sub too.

Reply:
@kszucs can you make those changes in another PR?

Reply:
@cpcloud sure

Reply:
Great!